spec/defines/rule_spec.rb

```diff
@@ -127,12 +127,91 @@
         end
 
         it { is_expected.to compile.with_all_deps }
-        it { is_expected.to contain_concat__fragment('INPUT-filter-consul').with_content("mod comment comment 'filter-consul' proto (tcp udp) dports (8301 8302) saddr @ipfilter((127.0.0.1)) ACCEPT;\n") }
+        it { is_expected.to contain_concat__fragment('INPUT-filter-consul').with_content("mod comment comment 'filter-consul' proto (tcp udp) mod multiport destination-ports (8301 8302) saddr @ipfilter((127.0.0.1)) ACCEPT;\n") }
```

Comment:
This seems like it changing the default behavior which would require a major version bump. Suggest leaving the default behavior alone.

Reply:
Oh quite the contrary, this actually fixes the multiport rule generation on older systems.
As state in the commit message:
```
use verbose multiport syntax for better compat

The dports/sports shortcut is only supported starting with ferm v2.5 which
was released very recently.
In order to support a wider range of distributions and ferm versions,
this commits switches to the more verbose version of the multiport
features.
```

Otherwise the generated ferm snippets containing a `dports` shortcut rule will cause ferm to fail and not generate any rules at all.

Reply:
I'd still bump the version
people out there might have been working around this bug, and now we might be [breaking their workaround](https://xkcd.com/1172/)

Reply:
fine with me, we can just do a `v5.0.0` release after this PR is merged